src/components/ChangePlan/ChangePlan.js

```diff
@@ -23,14 +40,27 @@ const ChangePlan = () => {
         <section className="dp-container">
           <div className="dp-rowflex">
             <div className="col-sm-12" style={{ textAlign: 'center' }}>
-              <h1>Compra un plan</h1>
-              <a href={planUrl} class="dp-button button-medium primary-green">
-                Comprar Plan
-              </a>
-              <br />
-              <br />
-              Información del plan Plan Id {planId} - DiscountId - {safeDiscountId} - PromoCode -{' '}
-              {safePromoId}
+              <Card>
+                <div className="dp-content-plans">
+                  <h3>{cardData.name}</h3>
+                  <p>{_('change_plan.description')}</p>
+                  <p>{cardData.descriptionPlan}</p>
+                  <CardPrice currency="US$">{cardData.price}</CardPrice>
+                  <CardAction url={cardData.action.url}>{cardData.action.text}</CardAction>
+                </div>
+              </Card>
+              <Card cssClass="dp-highlighthed">
```

Comment:
Since it is a common scenario, what do you think about having a flag?

```suggestion
              <Card highlighthed>
```

Reply:
but if had another css class name for another thing?

Reply:
Creo eso está relacionado con este comentario: https://github.com/FromDoppler/doppler-webapp/pull/1023/files#r467954360-permalink

Creo que hay formas comunes de arrastrar info general a los componentes.

Si está resaltado o no es algo específico, otras clases serían algo general. Creo que debería ser posible convinar ambas en algo así:

```html
<Card highlighthed className="my-custom-class">
```

Pero no se como se hace, creo que deberíamos investigarlo y aprenderlo.

Reply:
[Acá](https://stackoverflow.com/questions/32230635/passing-in-class-names-to-react-components#43029805) proponen hacerlo así:

```
export const ChildComponent = ({ className, children }) =>
  <div className={`some-css-className ${className}`}>
    {children}
  </div>
```

Pero no se si es al forma más simple o correcta.

Reply:
[Este paquete](https://www.npmjs.com/package/classnames) parece útil para hacer algo así:

```
const Card = ({ children, className, highlighthed, ...rest }) => {
  return <div
    class={classNames(
      {"dp-card": true, "highlighthed": !!highlighthed}, 
      className)}
    {...rest}>
    {children}
  </div>;
};
```

Reply:
buenisimo! ahi lo cambie